src/Core/Components/DataGrid/Columns/ColumnBase.razor.cs

```diff
@@ -128,6 +128,16 @@ public abstract partial class ColumnBase<TGridItem>
     /// </summary>
     protected FluentDataGrid<TGridItem> Grid => InternalGridContext.Grid;
 
+    /// <summary>
+    /// Get items.
```

Comment:
Can you give a different explanation of the property name?
`Gets a list of all elements included in the grid.`

Reply:
the items are calculated they are not all

Reply:
the items comes from here and it is not all the items

https://github.com/microsoft/fluentui-blazor/blob/cfb055ced298d26f83ebcb21ed31a636025db63c/src/Core/Components/DataGrid/FluentDataGrid.razor.cs#L512